srcpkgs/amule/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-master
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man
+	--enable-cas --enable-wxcas --enable-amule-daemon
+	--enable-amulecmd --enable-amule-gui --enable-alc
+	--enable-alcc --enable-webserver --disable-debug
+	--enable-optimize --enable-geoip --disable-upnp
+	--enable-fileview --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel wxWidgets-gtk3"
+makedepends="boost-devel zlib-devel wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+depends="wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+short_desc="Emule-like client for the eD2k and Kademlia networks"
+maintainer="hazayan <hazayan@hotmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/amule-project/amule"
+distfiles="https://github.com/amule-project/amule/archive/master.tar.gz"
+checksum="391d4fdee850ee4237e0302c11de3e21154dd16148bef5f7c35d3c96d92922ab"
+
+pre_configure() {
+	./autogen.sh
+}
+
+do_install() {
+	vbin src/amule
+	vbin src/amuled
+	vbin src/amulecmd
+	vbin src/amulegui
```

Comment:
Any reason to manually install the binaries instead of using the default `make install` target?

Reply:
Fixed.